src/pages/collators/dialogs/ClaimRewardsDialog.tsx

```diff
@@ -65,70 +60,74 @@ function ClaimRewardsDialog(props: Props) {
           if (status.isInBlock) {
             if (errors.length > 0) {
               const errorMessage = `Transaction failed with errors: ${errors.join(
-                "\n"
+                '\n',
               )}`;
               console.error(errorMessage);
-              toast(errorMessage, { type: "error" });
+              toast(errorMessage, { type: 'error' });
             }
           } else if (status.isFinalized) {
-            setLoading(true)
+            setLoading(true);
 
             if (errors.length === 0) {
-              setStep(ClaimStep.Success)
+              setStep(ClaimStep.Success);
             }
           }
-        }
+        },
       )
       .catch((error) => {
-        console.error("Transaction submission failed", error);
-        toast("Transaction submission failed", { type: "error" });
+        console.error('Transaction submission failed', error);
+        toast('Transaction submission failed', { type: 'error' });
         setLoading(false);
       });
-  }, [api, amount, createClaimRewardExtrinsic,
-    walletAccount, setLoading, setStep]);
-
+  }, [
+    walletAccount,
+    api,
+    amount,
+    createClaimRewardExtrinsic,
+    userRewardsBalance,
+  ]);
 
   const content = useMemo(() => {
     switch (step) {
       case ClaimStep.Confirm:
         return (
           <div className="rounded-lg bg-base-200 flex flex-col p-8 items-center w-fit center m-auto">
             <p className="flex">Amount</p>
-            <h1 className="flex text-4xl">{format(amount, tokenSymbol, true)}</h1>
```

Comment:
Hey @nejcm, do you have an auto formatter ? In that cases, which one? So we all agree in one with the same rules.
This causes that every single file is changed to a new format, leading to conflicts with lines that you didn't even touch. I can't approve this PR with so many unrelated lines changed